desktop/apps/auction2/components/page/Banner.js

```diff
@@ -62,7 +41,47 @@ export default function Banner (props) {
 }
 
 Banner.propTypes = {
+  BackgroundBanner: PropTypes.func.isRequired,
   auction: PropTypes.object.isRequired,
   isLiveOpen: PropTypes.bool.isRequired,
   liveAuctionUrl: PropTypes.string
 }
+
+const mapStateToProps = (state) => {
+  const {
+    auction,
+    isLiveOpen,
+    liveAuctionUrl
+  } = state.app
+
+  const { cover_image, name } = auction.toJSON()
+
+  const BackgroundBanner = ({ children }) => {
+    const imageUrl = get(cover_image, 'cropped.url', '')
+
+    return (
+      <div
+        className='auction2-banner-live-bg'
+        style={{ backgroundImage: `url('${imageUrl}')` }}
+        alt={name}
+      >
+        {children}
+      </div>
+    )
+  }
+
+  BackgroundBanner.propTypes = {
+    children: PropTypes.node
+  }
```

Comment:
Is it starting to get esoteric to pass in a component as mapped property? Maybe not, but just raising the question, since it's a pattern I haven't seen yet :)